src/pages/filter/components/filter-body/filter-body.component.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { filterAction } from '../../../../store/actions/filter.action';
+import { OverviewCheckboxes } from '../overview-checkboxes/overview-checkboxes.component';
+import { PetSizeCheckboxes } from '../petSize-checkboxes/petSize-checkboxes.component';
+import { SliderSection } from '../slider-section/slider-section.component';
+import { TimePerMonthCheckboxes } from '../timePerMonth-checkboxes/timePerMonth-checkboxes.component';
+import { Button } from '../button/button.component';
+import { Body } from '../../../comparison/interfaces/filter.interface';
+import './filter-body.scss';
+
+export const FilterBody: React.FC<Body> = () => {
+  const { register, handleSubmit, errors } = useForm();
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  const goBack = (): void => {
+    dispatch(filterAction({}));
```

Comment:
Passing empty object is not good practice. I would recommend to use `null` for such cases. Yes, it add additional logic, but it more clear that empty object.